pkg/endpoint/bpf.go

```diff
@@ -203,6 +204,9 @@ func (e *Endpoint) writeHeaderfile(prefix string, owner Owner) error {
 	// Always enable L4 and L3 load balancer for now
 	fw.WriteString("#define LB_L3\n")
 	fw.WriteString("#define LB_L4\n")
+	if option.Config.IsPolicyEnforcementInterfaceSet() {
+		fw.WriteString("#define POLICY_ENFORCEMENT_MODE\n")
```

Comment:
This seems to be duplicated with the following change in init.sh:
```
    OPTS+=" -DPOLICY_ENFORCEMENT_MODE"
```

If you intend for this to be a general node configuration option, then this should be placed into `node_config.h` from `writeNetdevHeader()`, and not configured per endpoint nor in the init script.